content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -84,45 +84,309 @@ If `--config` is provided and the values are not specified via the command line,
 defaults for the `KubeletConfiguration` version apply.
 In the above example, this version is `kubelet.config.k8s.io/v1beta1`.
 
+## Viewing the kubelet configuration
```

Comment:
what I am looking for when asking for /configz documentation is a clear statement on what that config represents:

1. Does it merge-in all the command line arguments into it?
2. Does it merge-in all the drop-in configuration files?
3.  Does it populate all the defaults?

In the example in this section, the feature gate consist of a single `"AllAlpha": false`. I wonder if it was configured by you explicitly or this is what it returns out of the box with nothing pre-configured?

Reply:
>what I am looking for when asking for /configz documentation is a clear statement on what that config represents:

>    Does it merge-in all the command line arguments into it?
>   Does it merge-in all the drop-in configuration files?
>  Does it populate all the defaults?

Do you want me to include this point under the Drop-in directory for kubelet configuration files? Otherwise, it may not make much sense without that background knowledge

Reply:
>In the example in this section, the feature gate consist of a single "AllAlpha": false. I wonder if it was configured by you explicitly or this is what it returns out of the box with nothing pre-configured?

I didn't configure the Kubelet config; it came out of the box.

Reply:
> > what I am looking for when asking for /configz documentation is a clear statement on what that config represents:
> 
> > Does it merge-in all the command line arguments into it?
> > Does it merge-in all the drop-in configuration files?
> > Does it populate all the defaults?
> 
> Do you want me to include this point under the Drop-in directory for kubelet configuration files? Otherwise, it may not make much sense without that background knowledge

it does make sense when we talk about arguments and such. If it were separate page about /configz, than it will definitely make sense and we can simply refer to it. If we don't have it as of now - simply mentiuoning that this is a final merged config will make the most sense.